eclair-core/src/main/scala/fr/acinq/eclair/io/Peer.scala

```diff
@@ -643,7 +642,7 @@ object Peer {
       channelKeyPath,
       dustLimit = nodeParams.dustLimit,
       maxHtlcValueInFlightMsat = nodeParams.maxHtlcValueInFlightMsat,
-      channelReserve = maxOf(Satoshi((nodeParams.reserveToFundingRatio * fundingAmount.toLong).toLong), nodeParams.dustLimit), // BOLT #2: make sure that our reserve is above our dust limit
+      channelReserve = Satoshi((nodeParams.reserveToFundingRatio * fundingAmount.toLong).toLong).max(nodeParams.dustLimit), // BOLT #2: make sure that our reserve is above our dust limit
```

Comment:
This could be made more readable using:
```suggestion
      (fundingAmount.toMilliSatoshi * nodeParams.reserveToFundingRatio).max(nodeParams.dustLimit), // BOLT #2: make sure that our reserve is above our dust limit
```